docs/sources/flow/tutorials/assets/generate.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# This script generates runt.sh script that pulls down the needed files and runs them.
```

Comment:
I think this comment needs to explain what's going on here a little more. None of us are bash experts, so I find it a little hard to follow. 

I think explaining what the "needed files" are would go a long way here.

I think we should also comment when this script would need to be re-run.